Makefile

```diff
@@ -33,6 +35,16 @@ ifeq ($(wildcard app/config/paypal_api.dev.yml),)
 endif
 	docker-compose run --rm --no-deps app ./bin/console app:validate:config app/config/config.dist.json app/config/config.dev.json
 
+# This needs the "drone" and "pass" cli tools to be installed on or local machine
+drone-ci:
+	@echo -n "GITHUB_TOKEN=" > var/.drone-secrets.txt
+	@pass wmde-fun/github-composer-ci-token >> var/.drone-secrets.txt
+	-@drone exec --secret-file var/.drone-secrets.txt --trusted .drone.yml ; \
+	RET=$$?; \
+	rm -f var/.drone-secrets.txt; \
+	if [ $$RET -eq 0 ]; then echo -e "\033[0;32mDrone CI passed\033[0m"; else echo -e "\033[0;31mDrone CI failed\033[0m"; fi; \
```

Comment:
1.  My terminal prints `-e` before the actual message.
      Removing `-e` after `echo` resolves this issue.
2.  Could you add `\n` before and after the message? 
      Like: `if [ $$RET -eq 0 ]; then echo "\033[0;32m\nDrone CI passed\n\033[0m"; else echo "\033[0;31m\nDrone CI failed\n\033[0m"; fi; \`

Reply:
One more thing:

![Screenshot from 2024-03-28 09-45-56](https://github.com/wmde/fundraising-application/assets/30310702/7bde09b0-2d6a-4d97-b8ec-d27d291a7e24)


Could we put `coverage.xml` in `git ignore` file? So that the issue shown in the screenshot can be resolved?

Reply:
Sure, you can put it in the `.gitignore file`

Reply:
![Screenshot from 2024-05-13 13-50-52](https://github.com/wmde/fundraising-application/assets/30310702/77af8539-1605-4829-bbda-7f8aef818181)

Proof: Terminal displays `-e` before printing the success or failure message.